AWBWApp.Game/UI/CameraControllerWithGrid.cs

```diff
@@ -34,6 +34,7 @@ public partial class CameraControllerWithGrid : Container
         private IBindable<bool> allowLeftMouseToDragUserOption;
         private IBindable<Colour4> baseMapColour;
         private IBindable<Colour4> baseGridColour;
+        private Bindable<bool> lockMapPosition;
```

Comment:
Should be an IBindable. IBindable means that it can't edit the underlining value, which is true in this case.